src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/common/ritual/trigger/ItemDropRitualTrigger.java

```diff
@@ -66,11 +66,11 @@ public static ItemDropRitualTrigger stack(ItemStack itemsStack) {
     }
 
     public static ItemDropRitualTrigger stacksExact(ItemStack... itemsStacks) {
-        return new ItemDropRitualTrigger(Arrays.stream(itemsStacks).<Ingredient>map(NBTIngredient::of).toList());
+        return new ItemDropRitualTrigger(Arrays.stream(itemsStacks).<Ingredient>map(StrictNBTIngredient::of).toList());
```

Comment:
Again, should probably be PartialNBTIngredient instead... though this can be discussed here

Reply:
Nope they can't be created from an ItemStack